.github/workflows/ci.yml

```diff
@@ -39,8 +39,38 @@ jobs:
             fi
           fi
 
+  # This job determines if this CI build is for a tagged release.
+  # It only runs when a tag is pushed to the `leanprover` repository.
+  # It sets `steps.set.outputs.release` to the tag, if the tag is "v" followed by a valid semver.
```

Comment:
Oh, I think you may have forgotten to update this doc? It does not set `steps.set.outputs.release` now but the various variables, correct? Or are you using `release` as a stand-in for all the different variables?